src/os/bluestore/BlueStore.cc

```diff
@@ -4914,10 +4914,10 @@ void BlueStore::_init_logger()
     "kd_l", PerfCountersBuilder::PRIO_USEFUL);
   b.add_time_avg(l_bluestore_state_deferred_queued_lat, "state_deferred_queued_lat",
     "Average deferred_queued state latency",
-    "sdql", PerfCountersBuilder::PRIO_USEFUL);
+    "dq_l", PerfCountersBuilder::PRIO_USEFUL);
   b.add_time_avg(l_bluestore_state_deferred_aio_wait_lat, "state_deferred_aio_wait_lat",
     "Average aio_wait state latency",
-    "dawl", PerfCountersBuilder::PRIO_USEFUL);
+    "diol", PerfCountersBuilder::PRIO_USEFUL);
```

Comment:
nit: how about daiowl?

Reply:
Nicks must be <= 4 characters (there is a ceph_assert() somewhere in the code that catches if you set the nick > 4 chars). With that, would you still recommend changing this up?

The reasoning behind `diol` was to emphasize the "io" part of the equation.